File: Indie Python Programming Course/step07/7.03/7.3.03.py
'''

Напишите функцию factorial, которая принимает на вход одно неотрицательное число, и возвращает значение факториала данного числа.

Нужно написать только определение функции factorial

factorial(3) => 6
factorial(1) => 1
factorial(0) => 1
factorial(5) => 120

# знак => обозначает return

Sample Input 1:

4

Sample Output 1:

24

Sample Input 2:

5

Sample Output 2:

120

'''

def factorial(x):
    pr = 1
    for i in range(2, x+1):
        pr *= i
    return pr


n=int(input())
print(factorial(n))

File: Diving into JavaScript: For Beginners/step07/7.03/7.3.03.js
/*

Напишите программу, которая запрашивает у пользователя ввод произвольной строки, 
а затем выводит каждый второй символ этой строки на отдельной строке в верхнем регистре.

Sample Input 1:

Жаба

Sample Output 1:

А
А

Sample Input 2:

годовщина

Sample Output 2:

О
О
Щ
Н

*/

const sentence = prompt();
for (let i = 1; i < sentence.length; i+=2) {
  console.log(sentence[i].toUpperCase());
}
